articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+
+### Machine Learning and Artificial Intelligence
+
+Machine learning is a technological tool that uses algorithmic interpretations. This has helped in obtaining and using data to make and interpret final decisions. Machine learning processes are also dependent on the subsets of artificial intelligence. Several [synthetic intelligence](https://www.wikiwand.com/en/Synthetic_intelligence) aspects and modern [virtual assistants](https://www.investopedia.com/terms/v/virtual-assistant.asp) are created to achieve this.
+
+Machine learning uses Artificial Intelligence (AI) techniques that assist in decision making. Conventional algorithms in AI aid in performing mathematical computation and predictions. Machine learning depends on fed structured data. Conventional algorithms aids in achieving data mining and predictive analytics in a search.
+
+Unlike machine learning, AI can use [unstructured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html) and still be able to function. Deep learning does mimic the human brain's workings. This is because it uses this data to perform the processing of data. Machine learning aids in the detection of objects, language translation, and decision making. Machine learning is also able to recognize speech and make prompt decisions. The AI makes this possible this allows in no aspect of human supervision.
+
+### Deep Learning
+
+Deep learning is a subset of machine learning. Deep learning uses networks that operate unsupervised and one that uses [structured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html). This is like the human brain structure as it operates as a subset of machine learning.
+
+This is to means that the algorithms used are complex and are from neural networks. This resemblance is like the layers with neural connections in the human brain.
+
+### How deep learning works
+
+Deep learning uses algorithms whose functionality is like that of machine learning. One major distinction on each layer is that it has a different interpretation of data. The [neural networks](https://news.mit.edu/2017/explained-neural-networks-deep-learning-0414) learn where data is fetched.
+
+The neurons in the network undertake two tasks. The first includes understanding which feature is optimal for performing a specific task. The second one involves classifying data after many iterations of the learning process. In case there is a mistake, the deep learning model's accuracy increases. Deep learning is, thus, a more complex and more capable version of machine learning.
+
+### Machine Learning vs. Deep Learning
+
+Deep learning differs from machine learning in some aspects. One major attribute is that deep learning learns from its mistakes. In most cases, this is achieved without human intervention. As for machine learning, there is a need for a human to correct the mistake made.
```

Comment:
Factual error: Both types of algorithms learn on their own once we feed them the data. But data cleaning, data pre-processing are essential steps in both types of algorithms. Consider mentioning the following point: Manual feature extraction is required for ML algorithms, whereas in the case of DL, there is no need for manually selecting features from the dataset.

Reply:
Noted. Thank you for the suggestion.